src/Uno.UWP/Microsoft/UI/Windowing/OverlappedPresenter.cs

```diff
@@ -77,13 +76,23 @@ public bool IsAlwaysOnTop
 
 	public bool HasTitleBar => _hasTitleBar;
 
-	public OverlappedPresenterState State => Native?.State ?? OverlappedPresenterState.Restored;
+	public OverlappedPresenterState State => Native?.State ?? _pendingState ?? OverlappedPresenterState.Restored;
```

Comment:
Doesn't having a pendingState here remove the need to do the same pendingState change in win32-specific code?